administrator/language/en-GB/plg_content_joomla.ini

```
@@ -8,6 +8,10 @@ PLG_CONTENT_JOOMLA_FIELD_CHECK_CATEGORIES_DESC="Check that categories are fully
 PLG_CONTENT_JOOMLA_FIELD_CHECK_CATEGORIES_LABEL="Check Category Deletion"
 PLG_CONTENT_JOOMLA_FIELD_EMAIL_NEW_FE_DESC="Email users if 'Send email' is on when there is a new article submitted via the Frontend."
 PLG_CONTENT_JOOMLA_FIELD_EMAIL_NEW_FE_LABEL="Email on New Site Article"
+PLG_CONTENT_JOOMLA_MAIL_NEWARTICLE_DESC="Sent when a new article is submitted via frontend."
+PLG_CONTENT_JOOMLA_MAIL_NEWARTICLE_TITLE="Content: New Article Submitted"
+PLG_CONTENT_JOOMLA_NEW_ARTICLE_BODY="A new Article has been submitted by {NAME} entitled {TITLE}."
```

```suggestion
PLG_CONTENT_JOOMLA_NEW_ARTICLE_BODY="A new article has been submitted by {NAME} entitled {TITLE}."
```
At least "article" should be lowercase in this sentence as it is not a name. But maybe @brianteeman could check and if necessary suggest other changes to this sentence. The `entitled` reads to me as if it would refer to the user and not to the article.

---

The suggested string is identical to the current string (just in a different place) COM_CONTENT_ON_NEW_CONTENT so I would say it is fine.

On the other hand use of the word "entitled" can be considered advanced english and could be replaced by either "titled" or "called"

---

I see. Thanks for reporting back.